hadoop-ozone/dist/src/main/compose/ozone-net-topology/network-config

```diff
@@ -0,0 +1,6 @@
+10.5.0.4	/rack1
```

Comment:
we may also pass the ip->location as a file and use the table mapping class like in the unit test.

Reply:
Xiaoyu, I thought about this before. It turns out that hostname and Ip address array still need to be kept in the code.  The code and data (table mapping file content) will be highly coherent. So I would prefer the current way.  What do you think?

Reply:
Sure. Will update the UT.